great_expectations/datasource/fluent/spark_datasource.py

```diff
@@ -70,6 +71,19 @@ class _SparkDatasource(Datasource):
     force_reuse_spark_context: bool = True
     persist: bool = True
 
+    @pydantic.validator("force_reuse_spark_context")
+    @classmethod
+    def _force_reuse_spark_context_deprecation_warning(cls, v: bool) -> bool:
+        if v is not None:
+            warnings.warn(
+                "force_reuse_spark_context is deprecated and will be removed in version 1.0. "
+                "In environments that allow it, the existing Spark context will be reused, adding the "
+                "spark_config options that have been passed. If the Spark context cannot be updated with "
+                "the spark_config, the context will be stopped and restarted with the new spark_config.",
+                category=RuntimeWarning,
```

Comment:
```suggestion
                category=DeprecationWarning,
```
https://docs.python.org/3/library/exceptions.html#DeprecationWarning